niceday-api/service/UserTrackersService.js

```diff
@@ -21,3 +22,30 @@ exports.setUserTrackerStatuses = (req, body) => new Promise((resolve, reject) =>
       reject(err);
     });
 });
+
+/**
+ * Get smoking tracker data.
```

Comment:
Maybe make this comment a bit more descriptive

Reply:
Check